### DISCUSSION TITLE
第5节课作业(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#372] [作业]
13班第五节课作业

**Comment by qingyin741** [comment]
![Snipaste_2024-01-16_14-42-52](https://github.com/InternLM/tutorial/assets/118804949/f9697280-708c-41b4-b4da-3060594abd8d)

![Snipaste_2024-01-16_14-55-38](https://github.com/InternLM/tutorial/assets/118804949/3d420cd2-294a-4f00-8cab-7a9ba3756ace)

![Snipaste_2024-01-16_15-22-43](https://github.com/InternLM/tutorial/assets/118804949/bcab6fff-943f-4ba9-bf73-6fb81d77b0f0)
![Snipaste_2024-01-16_15-35-31](https://github.com/InternLM/tutorial/assets/118804949/10a228b1-d382-4415-82ab-953a362efb50)

**Reply by SiliconSisyphus** [reply]
使用 LMDeploy 以本地对话,部署 InternLM-Chat-7B 模型，生成 300 字的小故事<img width="575" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/c47de9c9-818e-4409-8151-09df9f708178">

**Comment by qingtao12138** [comment]
第五课作业
https://docs.qq.com/doc/DREh5SnJVcXBrek9D

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/He1awmxLRiw3jskdwJscLPEqnGe?from=from_copylink

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135663659
